docs/conf.py

```diff
@@ -244,6 +243,7 @@
     "REPOS_FILE_BRANCH": repos_file_branch,
 }
 
+
```

Comment:
```suggestion
```

Reply:
Need to be 2 newlines because class definition follows